app/static/js/events/ticket_sales/attendees.js

```diff
@@ -0,0 +1,50 @@
+var $ticketFilterForm = $("#ticket-filter-form");
+var base_url = [location.protocol, '//', location.host, location.pathname].join('');
+
+$(".ticket-filter-open").click(function() {
+    $ticketFilterForm.show();
+    $ticketFilterForm.find("select").val("");
+});
+
+function make_request() {
```

Comment:
@niranjan94 I used conditional statements this time but the issue still persists. Maybe we can ignore it. There is not much to worry about. Please review it. Thanks!!